locales/base/translation.json

```diff
@@ -2614,7 +2614,8 @@
     },
     "poolFilters": {
       "openPools": "Open Pools",
```

Comment:
is this used any longer? if not, can we remove?

Reply:
Removed in ba933e128cc2dc5655a678f3534d6f669e9437b6!